docs/getting-started/run-applications.md

```diff
@@ -22,7 +22,7 @@ native application. In this getting started guide:
 * _Application Operators_ compose, deploy, and run application configurations.
 
 We'll play the roles of each of these team members as we deploy an application -
-Service Tracker - that consists of several services. One of these services, the
+Service Tracker - that consists of several micro services. One of these services, the
```

Comment:
I'd prefer to just say "services" without putting any comment on the size or shape of those services. There's a recent trend away from assuming all distributed systems are made of _micro_ services.

Reply:
ACK, I thought it's a typo.